src/conf_mode/nat64.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2023 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# pylint: disable=empty-docstring,missing-module-docstring
+
+import csv
+import json
+import os
+import re
+from ipaddress import IPv6Network
+
+from vyos import ConfigError, airbag
+from vyos.config import Config
+from vyos.configdict import dict_merge, is_node_changed
+from vyos.util import check_kmod, cmd, dict_search, run
+from vyos.xml import defaults
```

Comment:
Why import `defaults`? I see no reference to `<defaultValue>` node in your XML definitions.

Reply:
Almost certainly a copy-paste and me not understanding

Reply:
Actually, it looks like it's being used here https://github.com/vyos/vyos-1x/pull/1993/files#diff-301b6d4ad94f8356451b06c8710ea3b527e8dba33dc9727cdcb83997c05d8966R49

Should I just remove that `defaults` block entirely?

Reply:
If you do not consume any defaults, please drop it